xpdtools/tools.py

```diff
@@ -580,3 +580,30 @@ def inner(x, *args, **kwargs):
         return func(*args, **kwargs)
 
     return inner
+
+
+def avg_curvature(y, x, high_val):
+    """Computes the average of the curvature of a given signal past a
+    (currently) user provided high Q value, this will be used as the scalar
```

Comment:
don't say what this will be used for.  If it is just for that qoi it shouldn't be a separate function.  If it has more broad application, it should be its own function but the docstring should just describe what it does.